app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -17,4 +18,12 @@ class ReportsScreen(context: Context, attrs: AttributeSet) : LinearLayout(contex
       return
     }
   }
+
+  fun showReport(uri: URI) {
+    TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Maybe change your IDE settings to remove `//To change body of created functions use File | Settings | File Templates.`?

Reply:
Oh yes, I've forgotten to do this. I'll do it now.